src/main/java/uk/ac/ebi/eva/commons/models/converters/data/DBObjectToVariantAnnotationConverter.java

```diff
@@ -0,0 +1,234 @@
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.ac.ebi.eva.commons.models.converters.data;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.ObjectWriter;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import com.mongodb.BasicDBList;
+import com.mongodb.DBObject;
+import org.opencb.biodata.models.variant.annotation.ConsequenceType;
+import org.opencb.biodata.models.variant.annotation.ConsequenceTypeMappings;
+import org.opencb.biodata.models.variant.annotation.Score;
+import org.opencb.biodata.models.variant.annotation.VariantAnnotation;
+import org.opencb.biodata.models.variant.annotation.Xref;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.core.convert.converter.Converter;
+
+import java.util.LinkedList;
+import java.util.List;
+
+public class DBObjectToVariantAnnotationConverter implements Converter<DBObject, VariantAnnotation> {
+
+    public final static String CONSEQUENCE_TYPE_FIELD = "ct";
+    public static final String GENE_NAME_FIELD = "gn";
+    public static final String ENSEMBL_GENE_ID_FIELD = "ensg";
+    public static final String ENSEMBL_TRANSCRIPT_ID_FIELD = "enst";
+    public static final String RELATIVE_POS_FIELD = "relPos";
+    public static final String CODON_FIELD = "codon";
+    public static final String STRAND_FIELD = "strand";
+    public static final String BIOTYPE_FIELD = "bt";
+    public static final String C_DNA_POSITION_FIELD = "cDnaPos";
+    public static final String CDS_POSITION_FIELD = "cdsPos";
+    public static final String AA_POSITION_FIELD = "aaPos";
+    public static final String AA_CHANGE_FIELD = "aaChange";
+    public static final String SO_ACCESSION_FIELD = "so";
+    public static final String PROTEIN_SUBSTITUTION_SCORE_FIELD = "ps_score";
+    public static final String POLYPHEN_FIELD = "polyphen";
+    public static final String SIFT_FIELD = "sift";
+
+    public static final String XREFS_FIELD = "xrefs";
+    public final static String XREF_ID_FIELD = "id";
+    public final static String XREF_SOURCE_FIELD = "src";
+
+    public static final String CONSERVED_REGION_SCORE_FIELD = "cr_score";
+    public final static String SCORE_SCORE_FIELD = "sc";
+    public final static String SCORE_SOURCE_FIELD = "src";
+    public final static String SCORE_DESCRIPTION_FIELD = "desc";
+
+    public static final String CLINICAL_DATA_FIELD = "clinicalData";
+
+    private final ObjectWriter writer;
+
+    protected static Logger logger = LoggerFactory
+            .getLogger(uk.ac.ebi.eva.commons.models.converters.data.DBObjectToVariantAnnotationConverter.class);
```

Comment:
I think that intellij does that for you when you paste classes from other packages, in order to maintain as much as possible